Applications/Help/main.cpp

```diff
@@ -128,11 +128,10 @@ int main(int argc, char* argv[])
         auto buffer = file->read_all();
         StringView source { (const char*)buffer.data(), buffer.size() };
 
-        Markdown::Document md_document;
-        bool success = md_document.parse(source);
-        ASSERT(success);
+        auto md_document = Markdown::Document::parse(source);
+        ASSERT(md_document);
 
-        String html = md_document.render_to_html();
+        String html = md_document->render_to_html();
```

Comment:
I'm thinking we should drop the `md_document` after this line. C++ has no explicit way to drop local variables, but you can use scopes:

```cpp
String html;

{
    auto md_document = Markdown::Document::parse(source);
    ASSERT(md_document);
    html = md_document->render_to_html();
}
```